Assets/Scripts/Layers/Adapters/ChainOfResponsibility.cs

```diff
@@ -0,0 +1,134 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using UnityEngine.Events;
+using UnityEngine.Networking;
+
+namespace Netherlands3D.Twin
+{
+    public class ChainOfResponsibility : MonoBehaviour
```

Comment:
Can we rename this to DataTypeChain (or so)? This is not a re-usable ChainOfResponsibility implementation and could provide naming clashes or confusion

Reply:
Yes good idea 👍